app/views/helpers/currentObligationsHelper.scala.html

```diff
@@ -0,0 +1,134 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import implicits.ImplicitDateFormatter._
+
+@(obligations: models.reportDeadlines.ObligationsModel)(implicit messages: Messages, user: auth.MtdItUser[_])
+
+<section id="sub-heading-section">
+    <h2 id="page-sub-heading" class="heading-large"> @messages("obligations.sub-heading.v2") </h2>
```

Comment:
I don't think the h2 should be a part of this helper, it would be replaced by the tab's own text